### DISCUSSION TITLE
Bug fix of volume(odf)?

### DISCUSSION THREAD

**Discussion by MarcoWitte1** [mtex-toolbox/mtex#751] [Ask Anything]
Hi,

the new and bug fixed (?) version of volume(odf) gives now different results compared to texture volume calculations from EBSD orientations. 
Before the MTEX V5.5 bug fix there was a good agreement between volume fractions calculated directly from EBSD and calculated from an odf derived from that same EBSD data. Now the volume fractions from the odf are about 50% smaller.
Is there maybe something wrong with the bug fix?

Cheers
Marco

**Comment by ralfHielscher** [comment]
Hi Marco,

can you provide some more details, crystal, specimen symmetry, ...? Maybe some code?

Ralf.

**Reply by MarcoWitte1** [reply]
Hi Ralf,

when I was preparing my code for sending it to you, I already found the mistake that caused the problem! 
Somewhere I had used mean(...) instead of sum(...).  Sorry for bothering.

Cheers 
Marco
